<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVoiceSetupForm</class>
 <widget class="QFrame" name="CVoiceSetupForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>217</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gl_VoiceSetup">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_VatsimUdpPort">
     <property name="text">
      <string>VATSIM UDP Port:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_VoiceSetup">
     <property name="text">
      <string>Voice setup:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_VatsimUdpPort"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="wi_Override" native="true">
     <layout class="QHBoxLayout" name="hl_Override">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="cb_Override">
        <property name="text">
         <string>override</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignRight">
       <widget class="QPushButton" name="pb_SetDefaults">
        <property name="text">
         <string>set defaults</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_Override</tabstop>
  <tabstop>pb_SetDefaults</tabstop>
  <tabstop>le_VatsimUdpPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
